server/server.go

```diff
@@ -190,7 +194,7 @@ func (h *handler) handle(pkt *protocol.Packet) {
 	defer h.mtx.Unlock()
 	defer func() {
 		logRequestExecDuration(pkt.Operation.Opcode, start, resp.op.ErrorVal())
```

Comment:
It doesn't make sense why `ExecDuration` should include anything besides the actual execution of the operation. Why not keep it as a part of `makeRespondResponse` etc as it was before:
https://github.com/cloudflare/gokeyless/blob/34973625ac18b695a4735b0e045007b52408ef0d/server/server.go#L320-L333

Reply:
We're not done until we've sent back the request.

Reply:
But that is the purpose of `TotalDuration`, `ExecDuration` shouldn't be concerned with that. The previous version of the code didn't consider the sending bits either. And the actual metric seems to suggest the same:
https://github.com/cloudflare/gokeyless/blob/1c828f26e77404b5111574dbd29f26ef3ba46bd3/server/metrics.go#L21-L27

Reply:
Makes sense